user-guide/Advanced_Functionality/Databases/Configuring_dedicated_clustered_storage/Migrating_existing_systems/Migrating_the_general_database_to_Cassandra.md

```diff
@@ -28,6 +28,22 @@ The migration procedure has the following prerequisites:
 
 - The DataMiner Agents did not restart in the last 48 hours.
 
+> [!IMPORTANT]
```

Comment:
I think this warning may not be in the right location. To my knowledge, the wizard to migrate from MySQL to Cassandra is usually also used to install Cassandra itself, which means that it is not possible for the user to configure any Cassandra settings before they start the wizard.